adding pkce link, em dash clean up, sample code na removed, other nits (#2887)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/social-login/main/index.md

1. Click **Next**.
1. In the **General Settings** section, define the following:

    * **Name**: Enter a name for the Identity Provider in Okta.

>     * **Client ID**: Paste the generated client ID from your <StackSelector snippet="idp" noSelector inline /> application.

Reviewer: ```suggestion
    * **Client ID**: Paste the generated client ID from your <StackSnippet snippet="idp" inline /> application.
```

>     * **Client Secret**: Paste the generated client secret from your <StackSelector snippet="idp" noSelector inline /> application.

Reviewer: ```suggestion
    * **Client Secret**: Paste the generated client secret from your <StackSnippet snippet="idp" inline /> application.
```

>     * **Scopes**: Leave the defaults for a simple sign-in flow. You can also add more scopes. See <StackSelector snippet="scopes" noSelector inline />.

Reviewer: ```suggestion
    * **Scopes**: Leave the defaults for a simple sign-in flow. You can also add more scopes. See <StackSnippet snippet="scopes" inline />.
```

In the URL, replace `${yourOktaDomain}` with your org's base URL, and then replace the following values:

* `client_id`: Use the `client_id` value from your Okta app integration. This is not the `client_id` from the Identity Provider. For example, `0oawjqpb2wcUAWM8C0h7`.

> * `response_type`: Determines which flow is used. For the [Implicit](/docs/guides/implement-grant-type/implicit/main/) flow, this should be `id_token`. For the [Authorization Code](/docs/guides/implement-grant-type/authcode/main/) flow, this should be `code`.

Reviewer: ```suggestion
* `response_type`: Determines which flow is used. For the [Implicit](/docs/guides/implement-grant-type/implicit/main/) flow, use `id_token`. For the [Authorization Code](/docs/guides/implement-grant-type/authcode/main/) flow, use `code`.
```

> * `response_mode`: Determines how the authorization response should be returned. This should be `fragment`.

Reviewer: ```suggestion
* `response_mode`: Determines how the authorization response is returned. Use `fragment`.
```

<StackSnippet snippet="noemail" />

## Add <StackSnippet snippet="idp" inline /> to the Okta Sign-In Widget

> The [Okta Sign-In Widget](https://github.com/okta/okta-signin-widget) is an embeddable JavaScript Widget that reproduces the look and behavior of the standard Okta sign-in page. You can add a **Sign in with <StackSelector snippet="idp" noSelector inline />** button to the Widget by adding the following code to your Okta Sign-In Widget configuration. Replace `Your_IDP_ID` with the Identity Provider ID from your Identity Provider that you created in Okta in the [Create the Identity Provider in Okta](#create-the-identity-provider-in-okta) section.

Reviewer: ```suggestion
The [Okta Sign-In Widget](https://github.com/okta/okta-signin-widget) is an embeddable JavaScript Widget that reproduces the look and behavior of the standard Okta sign-in page. You can add a **Sign in with <StackSnippet snippet="idp" inline />** button to the Widget by adding the following code to your Okta Sign-In Widget configuration. Replace `Your_IDP_ID` with the Identity Provider ID from your Identity Provider that you created in Okta in the [Create the Identity Provider in Okta](#create-the-identity-provider-in-okta) section.
```
